q2_feature_table/plugin_setup.py

```diff
@@ -327,6 +327,36 @@
                 "on https://docs.qiime2.org for additional details."
 )
 
+plugin.methods.register_function(
+    function=q2_feature_table.filter_features_conditionally,
+    inputs={'table': FeatureTable[T1]},
+    parameters={'prevalence': Float % Range(0, 1),
+                'abundance': Float % Range(0, 1)
+                },
+    outputs=[('filtered_table', FeatureTable[T1])],
+    input_descriptions={
+        'table': 'The feature table from which features should be filtered.'
+    },
+    parameter_descriptions={
+        'abundance': ('The minimum relative abundance for a feature to be '
+                      'retained.'),
+        'prevalence': ('The minimum portion of samples that a feature '
+                       'must have a relative abundance of at least '
+                       '`abundance` to be retained.')
+    },
+    output_descriptions={
+        'filtered_table': 'The resulting feature table filtered by feature.'
+    },
+    name="Filter features from a table based on abundance and prevalence",
+    description=("Filter features based on the relative abundance in a "
+                 "certain portion of samples (i.e., features must have a "
+                 "relative abundance of at least `abundance` in at least "
+                 "`prevalence` number of samples). Any samples with a "
+                 "frequency of zero after feature filtering will also be "
+                 "removed.")
+)
```

Comment:
I think it would be more user friendly if the order of the parameters (and consequently, the order in which they are presented to the user) matched up with the order in which they are discussed in the description.  For example, if we register the `abundance` parameter first, we would get:

```Usage: qiime feature-table filter-features-conditionally [OPTIONS]

  Filter features based on the relative abundance in a certain portion of
  samples (i.e., features must have a relative abundance of at least
  `abundance` in at least `prevalence` number of samples). Any samples with
  a frequency of zero after feature filtering will also be removed.

Inputs:
  --i-table ARTIFACT FeatureTable[Frequency¹ | RelativeFrequency² |
    PresenceAbsence³ | Composition⁴]
                       The feature table from which features should be
                       filtered.                                    [required]
Parameters:
  --p-abundance PROPORTION
    Range(0, 1)        The minimum relative abundance for a feature to be
                       retained.                                    [required]
  --p-prevalence PROPORTION
    Range(0, 1)        The minimum portion of samples that a feature must
                       have a relative abundance of at least `abundance` to be
                       retained.                                    [required]
```
instead of this:

```
Usage: qiime feature-table filter-features-conditionally [OPTIONS]

  Filter features based on the relative abundance in a certain portion of
  samples (i.e., features must have a relative abundance of at least
  `abundance` in at least `prevalence` number of samples). Any samples with
  a frequency of zero after feature filtering will also be removed.

Inputs:
  --i-table ARTIFACT FeatureTable[Frequency¹ | RelativeFrequency² |
    PresenceAbsence³ | Composition⁴]
                       The feature table from which features should be
                       filtered.                                    [required]
Parameters:
  --p-prevalence PROPORTION
    Range(0, 1)        The minimum portion of samples that a feature must
                       have a relative abundance of at least `abundance` to be
                       retained.                                    [required]
  --p-abundance PROPORTION
    Range(0, 1)        The minimum relative abundance for a feature to be
                       retained.                                    [required]
```

The easiest solution would be to modify the description to match up with the current ordering.  Alternatively, we can change the order of the parameters wherever they appear to match up with the docstring.  Whichever solution you prefer is fine with me!  What do you think?